catalog/security/security-app/src/main/resources/features.xml

```diff
@@ -21,9 +21,16 @@
         <bundle>mvn:org.codice.alliance.security/banner-marking/${project.version}</bundle>
     </feature>
 
+    <feature name="security-defaultsecurityattributevalues" install="auto" version="${project.version}"
+             description="Plugin to mark unmarked metacards with the system high security markings.">
+        <feature>catalog-plugin-defaultsecurityattributevalues</feature>
+        <configfile finalname="/etc/ddf.catalog.metacard.security.defaultsecurityattributevaluesplugin.config" override="true">
+            mvn:org.codice.alliance.security/security-app/${project.version}/config/defaultsecurityattributevalues
+        </configfile>
+    </feature>
+
     <feature name="security-app" install="auto" version="${project.version}"
              description="The Alliance Security App provides features to enhance security. ::Alliance Security">
         <feature prerequisite="true">catalog-app</feature>
-        <feature>banner-marking</feature>
```

Comment:
I think the `banner-marking` feature should be marked `install="auto"` with a prereq on `security-app` so it is always on when `security-app` is installed. Thoughts @jlcsmith?